app/models/concerns/hyrax/user.rb

```diff
@@ -24,6 +24,7 @@ module Hyrax::User
 
     scope :guests, ->() { where(guest: true) }
     scope :registered, ->() { where(guest: false) }
+    scope :not_batch_or_audit_user, -> { where("#{Devise.authentication_keys.first} not in (?)", [::User.batch_user_key, ::User.audit_user_key]) }
```

Comment:
What would you think renaming this something like `:without_system_accounts` so that we can add other accounts to this in the future without touching the places in the code that use this scope?